ATTRIBUTIONS/wasabi-attribution.xml

```diff
@@ -611,4 +611,4 @@
          </downloadUrls>
       </dependency>
    </dependencies>
-</attributionReport>
```

Comment:
I think if we just check out the file and the difference will be deleted.  But it is just a minor issue.